app/staging/domain/om-workload.js

```diff
@@ -1,9 +1,10 @@
 class OmWorkload {
-  constructor (casesSummary, courtReports, instReports, caseDetails) {
+  constructor (casesSummary, courtReports, instReports, caseDetails, workloadId) {
     this.casesSummary = casesSummary
     this.courtReports = courtReports
     this.instReports = instReports
     this.caseDetails = caseDetails
+    this.workloadId = workloadId
```

Comment:
The staging schema doesn't have a workload table and the app workload won't have been created yet so the workloadId won't exist when we create any staging domain object.